src/java/relex/RelationExtractor.java

```diff
@@ -481,8 +489,8 @@ public static void main(String[] args)
 		}
 		catch (Exception e)
 		{
-			System.err.println("Unrecognized parameter.");
-			System.err.println(callString);
+			logger.error("Unrecognized parameter.", e);
+			logger.error(callString);
```

Comment:
Minor, but it will interpret ```{}``` in callString as variable placeholder, so for safety should be:

    logger.error("{}", callString)

Reply:
Updated the code.